addons/mail/static/src/models/attachment/attachment.js

```diff
@@ -215,9 +215,12 @@ function factory(dependencies) {
             } else if (!this.mimetype) {
                 return clear();
             }
+            if (this.mimetype.includes('text/')) {
```

Comment:
I think the intent here is not really "includes" but "starts with"? Probably best with indexOf === 0